### DISCUSSION TITLE
🚨⚠️PSA📢: Headless client ONLY works if ACE Headless is enabled ✅➡️

### DISCUSSION THREAD

**Discussion by Vdauphin** [Vdauphin/HeartsAndMinds#1339] [Announcement]
# 👋 Hey server administrators! 🧑‍🔧

Since the 1.22 update, H&M doesn't balance AI 🤖 between headless client. Instead, the new version just let the [ACE Headless system](https://ace3.acemod.org/wiki/framework/headless-framework.html) manage everything.
This allow H&M to use ACE headless features like handling from 1 to 3 headless client at the same time and manage efficiently AI between headless.

So:
### You ➡️**MUST**⬅️ enable ✅ ACE Headless through CBA Setting server side 🖥️.

If not, headless will do nothing 💤

_Have fun_ ❤️🧠

**Comment by OverlordZorn** [comment]
For people who are new to Headless Clients in general, what CBA settings would you recommend for Ace Headless?

**Reply by Vdauphin** [reply]
You just need to enable it
I didn't really needed to change other setting, may be ask in ACE Discord for more input
